test/development/acceptance-app/hydration-error.test.ts

```diff
@@ -146,7 +146,7 @@ createNextDescribe(
 
       await cleanup()
     })
-    // TODO: does not throw error, throw to show warning in overlay or remove this test
+    // TODO-APP: Decide if thesee logs should be displayed somewhere. They don't throw, just logs.
```

Comment:
nit: small typo on "these"
```suggestion
    // TODO-APP: Decide if these logs should be displayed somewhere. They don't throw, just logs.
```

Reply:
Thanks!